metadata-etl/src/main/resources/jython/HiveExtract.py

```diff
@@ -303,6 +300,7 @@ def run(self, schema_output_file, sample_output_file=None, hdfs_map_output_file=
 
     schema_json_file = open(schema_output_file, 'wb')
     os.chmod(schema_output_file, 0666)
```

Comment:
Let's use proper constants instead of 0666 here: https://docs.python.org/2/library/os.html#os.chmod